scripts/start.sh

```diff
@@ -30,6 +30,13 @@ else
   LETSENCRYPT_WILDCARD="false"
 fi
 
+if [ "$LETSENCRYPT_WILDCARD_DOMAIN_SAN" = "true" ] && [ "$LETSENCRYPT_WILDCARD" = "true" ]; then
+  echo "INFO: Main domain SAN requested and will be created at the same time"
+  LETSENCRYPT_WILDCARD_DOMAIN_SAN="true"
+else 
+  LETSENCRYPT_WILDCARD_DOMAIN_SAN="true"
```

Comment:
don't see the logic if the outcame is always LETSENCRYPT_WILDCARD_DOMAIN_SAN="true" ...

Reply:
I made a mistake! I'll fix it.